layerindex/tools/import_layer.py

```diff
@@ -180,6 +187,22 @@ def github_api_call(path):
 
     return (json_data, owner_json_data)
 
+def get_layer_type_choices():
+    """
+    Return help string and choices for --type.
+    """
+    from layerindex.models import LayerItem
+    help_str = "Specify layer type."
+    choices = []
+    for i in LayerItem.LAYER_TYPE_CHOICES:
+        key, description = i
+        help_str += ' %s: %s,' % (key, description)
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears that you are concatenating a string inside a loop. Concatenating immutable sequences will always results in a new object. This means that building up a sequence by repeated concatenation will have a quadratic runtime cost in the total sequence length. For efficiency and to get a linear runtime cost, consider appending values to a list and concatenating that list using [str.join()](https://docs.python.org/3/library/stdtypes.html#str.join) or [bytes.join()](https://docs.python.org/3/library/stdtypes.html#bytes.join) method.

[Learn more](https://docs.python.org/3/faq/programming.html#what-is-the-most-efficient-way-to-concatenate-many-strings-together)